tests/ttnn/unit_tests/operations/eltwise/backward/utility_funcs.py

```diff
@@ -11,6 +11,32 @@
 )
 
 
+def data_gen_with_range_batch_norm(
+    input_shapes,
+    low,
+    high,
+    device,
+    is_input=False,
+    required_grad=False,
+):
+    assert high > low, "Incorrect range provided"
+    torch.manual_seed(213919)
+    channels = input_shapes[1]
+    size = input_shapes if is_input else channels
+    pt_tensor = torch.rand(size, requires_grad=required_grad).bfloat16() * (high - low) + low
+    reshaped_tensor = pt_tensor
+    if not is_input:
+        reshaped_tensor = pt_tensor.view(1, channels, 1, 1).expand(1, channels, 32, 32)
```

Comment:
```suggestion
    reshaped_tensor = pt_tensor if is_input else pt_tensor.view(1, channels, 1, 1).expand(1, channels, 32, 32)
```
I think this would make it a lot less confusing to follow. I thought `reshaped_tensor` was unnecessary until I noticed `pt_tensor` in the return statement.